static/src/javascripts/projects/common/modules/article/space-filler.ts

```diff
@@ -23,6 +23,12 @@ class SpaceFiller {
 		const insertNextContent = () =>
 			findSpace(rules, options)
 				.then((paragraphs: HTMLElement[]) => writer(paragraphs))
+				.then(() => {
+					if (options?.debug) {
+						const event = new Event('adverts-created');
```

Comment:
Should this be a `CustomEvent`? I think the main difference is it is doesn't inherit any behaviour from regular DOM events.

Reply:
Yep. Thanks!